pkgs/development/python-modules/pyqtwebengine/default.nix

```diff
@@ -44,8 +46,17 @@ in buildPythonPackage rec {
     qtbase
     qtsvg
     qtwebengine
+    qtwebchannel
+    qtdeclarative
   ];
 
+  # TODO: figure out why the env hooks aren't adding these inclusions automatically
```

Comment:
Only the `/include` is added , not any sub paths IIRC

Many of the includes in https://github.com/search?q=%2F%23Include.%2BQtWebChannel%2F&type=code have a / so I don't know if they're supposed to be added to it.